src/org/javarosa/core/model/CoreModelModule.java

```diff
@@ -41,7 +41,7 @@
             "org.javarosa.core.model.data.LongData",
             "org.javarosa.core.model.data.MultiPointerAnswerData",
             "org.javarosa.core.model.data.PointerAnswerData",
-            "org.javarosa.core.model.data.SelectMultiData",
+            "org.javarosa.core.model.data.MultipleItemsData",
```

Comment:
Do you need to also keep SelectMultiData here? Do instances of that class get serialized?

Reply:
Hmm, good question, @dcbriccetti. They do. `SelectMultiData` is now a subclass of `MultipleItemsData` but if `SelectMultiData` is serialized, it does need to be here despite that, right?

Reply:
Fixed.